pkgs/applications/misc/pipewire-screenaudio/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gawk
+, hexdump
+, jq
+, pipewire
+, psmisc
+}:
+stdenv.mkDerivation rec {
+  pname = "pipewire-screenaudio";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "IceDBorn";
+    repo = "pipewire-screenaudio";
+    rev = version;
+    hash = "sha256-IfPW0qmIUMIuevMLolYyKpYMBiiBG1OJA7/Wtxp+EzM=";
+  };
+
+  buildInputs = [
+    gawk
+    hexdump
+    jq
+    pipewire
+    psmisc
+  ];
+
+  postInstall = ''
+    runHook preInstall
```

Comment:
Post install does not need the runHooks. Also, can you use substituteInPlace instead of sed?

Reply:
Like this?
```  
postInstall = ''
  mkdir -p $out/lib/out
  install -Dm755 native/connector/pipewire-screen-audio-connector.sh $out/lib/connector/pipewire-screen-audio-connector.sh
  install -Dm755 native/connector/virtmic.sh $out/lib/connector/virtmic.sh
  
  # Firefox manifest
  substituteInPlace native/native-messaging-hosts/firefox.json \
     --replace /usr/lib/pipewire-screenaudio $out/lib
  install -Dm644 native/native-messaging-hosts/firefox.json $out/lib/mozilla/native-messaging-hosts/com.icedborn.pipewirescreenaudioconnector.json
'';
```